src/Features/Core/Portable/LanguageServices/SymbolDisplayService/SymbolDescriptionGroups.cs

```diff
@@ -17,6 +17,7 @@ internal enum SymbolDescriptionGroups
         AnonymousTypes = 1 << 4,
         Exceptions = 1 << 5,
         Captures = 1 << 6,
-        All = MainDescription | AwaitableUsageText | Documentation | TypeParameterMap | AnonymousTypes | Exceptions | Captures
+        ReturnsDocumentation = 1 << 7,
+        All = MainDescription | AwaitableUsageText | Documentation | TypeParameterMap | AnonymousTypes | Exceptions | Captures | ReturnsDocumentation,
```

Comment:
How does this work given this is an enum? Anybody compiling with an older library's "All" will still get the older behavior, right? Do we need to make this a sentinel value of some kind and deal with it in the API itself?

(This isn't really a _new_ problem, but still might have some impact to whatever is going on here.)

Reply:
The impact of the value change is minimal (if any) because it's an `internal` enum.